base/strings/basic.jl

```diff
@@ -403,6 +403,25 @@ julia> lowercase("STRINGS AND THINGS")
 lowercase(s::AbstractString) = map(lowercase, s)
 
 """
+    titlecase(s::AbstractString)
+
+Returns `s` with all initial characters converted to titlecase.
+
+```jldoctest
+julia> titlecase("the julia programming language")
+"The Julia Programming Language"
+```
+"""
+function titlecase(s::AbstractString)
+    words = split(lowercase(s))
```

Comment:
This is the behaviour I know from e.g. Python. The question is whether we want this.

This might be more useful

```julia
julia> str = "ISS - international space station"
julia> titlecase(str)
"ISS - International Space Station"
```

than this

```julia
julia> str = "ISS - international space station"
julia> titlecase(str)
"Iss - International Space Station"
```